lnbits/decorators.py

```diff
@@ -29,20 +30,20 @@ def __init__(
         self._key_type = "invoice"
         self._api_key = api_key
         if api_key:
-            self.model: APIKey = APIKey(
+            key = APIKey(
                 **{"in": APIKeyIn.query},
                 name="X-API-KEY",
                 description="Wallet API Key - QUERY",
             )
         else:
-            self.model: APIKey = APIKey(
+            key = APIKey(
                 **{"in": APIKeyIn.header},
                 name="X-API-KEY",
                 description="Wallet API Key - HEADER",
             )
-        self.wallet = None
+        self.model: APIKey = key
```

Comment:
You set it only if no error occurs. Before it was set either way. Does that matter?

Reply:
it is not about the error, mypy doesnt like defining attribute twice.
```lnbits/decorators.py:39: error: Attribute "model" already defined on line 33```